source-academy/frontend
# Fix Run Button On Mobile

### Description

Closes #2839 

On the mobile version of source academy, pressing the run button would not eval the code. Fixed the code such that on a tab change, this action would trigger the mobile tab change callback which would run the code. Due to the sagas change, the stepper and cse machine does not work on mobile either, would be creating another pull request to resolve this.

### Type of change


### How to test

On the mobile version of source academy, press the run button to confirm that the code would now be evaluated

<img width="375" alt="image" src="https://github.com/source-academy/frontend/assets/34702990/5cd20758-1789-4f55-bc6d-c0ff48411e9a">

### Checklist
